process/economics/economicsData.go

```diff
@@ -179,11 +183,16 @@ func convertValues(economics *config.EconomicsConfig) (*EconomicsData, error) {
 func checkValues(economics *config.EconomicsConfig) error {
 	if isPercentageInvalid(economics.RewardsSettings.LeaderPercentage) ||
```

Comment:
indeed these are not enough.
But there are different criterias as the leader percentage is applied to the accumulated fees in the proposed block, whereas the developerPercentage is applied to the transaction fee calling the speciffic SC, so also not part of the total. The community percentage is applied to the total.

Due to your comment I checked a bit and found a bug in the accumulated fees and what the Leader receives. The accumulated fees are considered entirely when computing the rewards per block, but then again the leader receives in addition the leader percentage -> leader percentage should have been subtracted before computing the rewards per block, otherwise we are giving that portion twice.

Reply:
great find :+1: